pages/news-and-views/_slug/index.vue

```diff
@@ -12,37 +14,57 @@ export default {
 		const linkList = store.state.posts.map(({ slug }) => slug)
 		return linkList.includes(params.slug)
 	},
-	head () {
-		return {
-			title: this.Title
-		}
-	},
+	// head () {
+	// 	return {
+	// 		title: this.Title
+	// 	}
+	// },
 	components: {
 		PostArticle
 	},
 	data () {
 		return {
 		}
 	},
-	computed: {
-		links () {
-			return this.$store.state.posts.map(({ slug }) => slug)
-		},
-		linkNames () {
-			return this.links.map(x => x.replace(/-/g, " "))
-		},
-		Title () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).title
-		},
-		Picture () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).picture
-		},
-		Content () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).content
-		},
-		Tags () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).tags
-		}
+	// computed: {
+	// 	links () {
+	// 		return this.$store.state.posts.map(({ slug }) => slug)
+	// 	},
+	// 	linkNames () {
+	// 		return this.links.map(x => x.replace(/-/g, " "))
+	// 	},
+	// 	Title () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).title
+	// 	},
+	// 	Picture () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).picture
+	// 	},
+	// 	Content () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).content
+	// 	},
+	// 	Tags () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).tags
+	// 	}
+	// },
+	asyncData (context) {
+		return axios.get(`${Config.wpDomain}${Config.apiBase}posts`).then((response) => {
+			const res = response.data.filter(x => x.slug === context.params.slug)[0]
+			let picture = "https://wecount.inclusivedesign.ca/wp-content/uploads/2019/10/We-Count-logos_colour-and-bw-01.png"
```

Comment:
A couple of thoughts:
1. Gather and define all default values at the top of the code for easy lookup;
2. Rename this variable to "placeHolderPicture" or "defaultPicture", change line 63 to `picture: picture || defaultPicture`, which means, use the default preview picture if there isn't one for this post. This will be able to also handle the case when `pic.source_url` is `null` or empty.